src/oal_agent/security/sandboxing.py

```
@@ -4,11 +4,32 @@
+        The sanitized, absolute path.
+
+    Raises:
+        ValueError: If the sanitized path attempts to escape the base_path.
+    """
+    # Resolve to real paths to handle symlinks
+    real_base_path = os.path.realpath(base_path)
+    real_target_path = os.path.realpath(os.path.join(base_path, target_path))
+
+    # Ensure the target path is within the base path
+    if not real_target_path.startswith(real_base_path):
+        raise ValueError(f"Path {target_path} attempts to escape base path {base_path}")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)